src/index.html

```diff
@@ -9,6 +9,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+         src="/layout_moyo-header/src/images/logo.png"
+         alt="logo"
+         class="logo-images">
```

Comment:
Need to fix it everywhere
```suggestion
        <img
          src="/layout_moyo-header/src/images/logo.png"
          alt="Moyo logo"
          class="logo-images"
        >
```